main.cpp

```diff
@@ -73,5 +461,36 @@ Purpose:  This project continues developing Project3.
 #include <iostream>
 int main()
 {
+    Dishwasher dishwasher;
+    dishwasher.dryDishes();
+    dishwasher.alarm();
+
+    dishwasher.dishes.tarnish(true);
+    dishwasher.cleanDishes(dishwasher.dishes);
+    dishwasher.cleanDishes(dishwasher.dishes);
```

Comment:
This is a common issue with new programmers or programmers who are coming from garbage-collected languages (python, C#).
When you pass this type into this function, the instance that is used inside this function is NOT the same instance that was passed in.
That means any changes you make to the instance being used INSIDE the function are NOT kept when this function ends.
the C++ concept of References and Pointers has not been covered at this point in the course. If you wanted this to work as you are most likely expecting it to work, you would need to use references or pointers.
I don't let people use coding concepts in projects until after they've been covered in the course.

What you should do here is declare an instance of `Dishwasher::Dishes` before line 469, modify some of its properties and pass that instance to the `Dishwasher::cleanDishes()` function.
- [ ] revise